string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,36 @@
+/* exported titleCase */
+
+function titleCase(title) {
+  var lowerCased = title.toLowerCase();
+  var lowerCasedList = ['and', 'or', 'nor', 'but', 'a', 'as', 'at', 'by', 'for', 'in', 'of', 'on', 'per', 'to', 'the'];
+  var words = lowerCased.split(' ');
+  words[0] = words[0].charAt(0).toUpperCase() + words[0].substring(1);
+  if (words[0].charAt(0) === 'J') {
+    words[0] = 'JavaScript:';
+    words[1] = 'The';
+  }
```

Comment:
This should work for _any_ book title, including:
- Book titles where the first word starts with a `'J'`, but isn't the word `'JavaScript'` (ex: `'Julius Caesar'`)
- Book titles that start with `'JavaScript'` where `'The'` isn't the second word (ex: `'JavaScript and JQuery: Interactive Front-End Web Development'`)

---

![image](https://user-images.githubusercontent.com/16143771/150581737-134c70b6-2b35-4088-8adf-b8cd5c66f9f8.png)